internal/repotools/release/calculate.go

```diff
@@ -63,14 +71,25 @@ func Calculate(finder ModuleFinder, tags git.ModuleTags, config repotools.Config
 				log.Fatalf("failed to get git changes: %v", err)
 			}
 
-			hasChanges, err = gomod.IsModuleChanged(moduleDir, repositoryModules[moduleDir], changes)
+			subModulePaths := repositoryModules[moduleDir]
```

Comment:
Can you give an example of what can be considered a subModulePath?

Reply:
For example when you look up the root module, all other modules in the repository like each service client path etc is listed in this slice so that it is ignored when looking at changes.